kalite/project/wsgi.py

```diff
@@ -0,0 +1,34 @@
+"""
+WSGI config for kalite.project
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.8/howto/deployment/wsgi/
```

Comment:
No wait yeah so I just left that module comment from Django 1.8's wsgi.py template, I didn't actually write it... but if you look below it makes sense!